tests/network/probes.go

```diff
@@ -231,6 +223,27 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 	})
 })
 
+func startGuestAgent(vmi *v1.VirtualMachineInstance) error {
+	const startAgent = "start"
+	return guestAgentOperation(vmi, startAgent)
+}
+
+func stopGuestAgent(vmi *v1.VirtualMachineInstance) error {
+	const stopAgent = "stop"
+	return guestAgentOperation(vmi, stopAgent)
+}
+
+func guestAgentOperation(vmi *v1.VirtualMachineInstance, startStopOperation string) error {
+	if startStopOperation != "start" && startStopOperation != "stop" {
```

Comment:
Since you're using hardcoded `"start"` and `"stop"` here, I would move the `const startAgent = "start"` and `const stopAgent = "stop"` to a top level so they can be reused